website/docs/create-workflow.md

```diff
@@ -10,30 +10,67 @@ Click on the **Schedule your first workflow** button on the home page to get sta
 
 ### 1. Choose Target Agent
 
-You have to select a cluster type. There are two types of clusters:
+This is the first step in workflow creation. In this step, you can select a target agent where the chaos workflow will be scheduled. These agents consist the CRDs and the required resources to run a chaos workflow.
+While installing the Litmus Portal, a default agent named **Self Cluster** is created.
 
-- Internal Cluster (Self Cluster)
-- External Clusters that are available on the targets screen.
-
-After Selecting the cluster, you can continue by clicking on **Select and Continue**.
+After Selecting the agent, you can continue by clicking on **Next** button.
 
 ```
 Note:
- You may have to wait for the cluster to be ready, after which you can move forward by again clicking on “Select and
- Continue”. Newly created users by the admin can’t schedule a workflow, As non-admin users, you will  get a message  ‘No Cluster Registered With Your Project ID, Please Wait…’   if you try to create a workflow.
+ You may have to wait for the agent to be up and ready, after which you can move forward by again clicking on “Next”. Newly created users by the admin can’t schedule a workflow, As non-admin users, you will  get a message  ‘No Cluster Registered With Your Project ID, Please Wait…’   if you try to create a workflow.
```

Comment:
You can use `>` instead of ```

The line `Newly created users by the admin can’t schedule a workflow` can be restructured as `Newly created users by the admin won't have any agent connected and this can’t schedule a workflow`